src/dev/sdram.h

```diff
@@ -55,19 +55,19 @@ SECTIONS
 */
 #ifndef RAM_AS4C16M16SA_H
 #define RAM_AS4C16M16SA_H /**< & */
-#ifdef __cplusplus
-extern "C"
-{
-#endif
 #include <stdint.h>
 #include "daisy_core.h"
+extern "C"
+{
+#include "util/hal_map.h"
+}
```

Comment:
It looks like these can move inside the implementation file.

afaik, the hal map files have not had to be included in any headers yet, and I'd like to keep it that way. It prevents any external reliance on the HAL itself unless the user explicitly includes it.